app/server-entry.js

```diff
@@ -6,7 +6,7 @@ export default ssrContext => {
   const s = dev && Date.now()
 
   return new Promise((resolve, reject) => {
-    const { app, router, store } = createApp()
+    const { app, router, store } = createApp(ssrContext.req)
```

Comment:
Forgive my noobness (I don't use GitHub much), but how do I make that change to the current pull request?